irohad/torii/query_service.hpp

```diff
@@ -57,8 +59,15 @@ namespace torii {
         pb_query_response_factory_;
     std::shared_ptr<iroha::torii::QueryProcessor> query_processor_;
 
-    std::unordered_map<std::string, iroha::protocol::QueryResponse &>
+    std::unordered_map<std::string, iroha::protocol::QueryResponse&>
```

Comment:
It will not work other way, with copy

Reply:
Why it is not work?

Reply:
Because for GRPC you need to answer by reference

Reply:
It's better to pass pointer imo

Reply:
Changing it is quite painful and, imo, does not have any relation to current pr. If we think we should do it it will be better to create a separate pr.